src/recon_buildblock/ForwardProjectorByBin.cxx

```diff
@@ -277,7 +273,7 @@ forward_project(RelatedViewgrams<float>& viewgrams,
      iter != viewgrams.end();
      ++iter)
       {
-    ViewSegmentNumbers vs(iter->get_view_num(), iter->get_segment_num());
+    ViewSegmentTOFNumbers vs(iter->get_view_num(), iter->get_segment_num());
```

Comment:
this would probably need a TOF index. I think we can make this transparant by adding a `Viewgram::get_view_segment_tof_nums()` (but see main comment for name), and call that here. Then we don't need to know this here.